packages/core/src/transport/eventBridge.ts

```diff
@@ -28,7 +29,7 @@ export function getEventBridge<T, E>() {
       return JSON.parse(eventBridgeGlobal.getCapabilities?.() || '[]') as BridgeCapability[]
     },
     getPrivacyLevel() {
-      return eventBridgeGlobal.getPrivacyLevel?.()
+      return eventBridgeGlobal.getPrivacyLevel?.() as DefaultPrivacyLevel
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ Can we make the `DatadogEventBridge`'s type better instead? (line 10)

```diff
-  getPrivacyLevel?(): string
+  getPrivacyLevel?(): DefaultPrivacyLevel